doc/requirements-docs.txt

```diff
@@ -1,40 +1,44 @@
 # these are most specified that usual, because Sphinx docs seem to be quite fragile
-Sphinx==4.4.0
+# limited <7 by sphinx-rtd-theme at the moment.
+# sphinx-rtd-theme requires docutils 0.18 but sphinx dropped that support in 6.0.0
+Sphinx==5.3.0
```

Comment:
Ok I resolved that problem but have a new one:

```
ERROR: Cannot install -r requirements-docs.txt (line 16), -r requirements-docs.txt (line 7) and sphinx==5.3.0 because these package versions have conflicting dependencies.

The conflict is caused by:
    sphinx 5.3.0 depends on docutils<0.20 and >=0.14
    nbsphinx 0.9.2 depends on docutils
    sphinx-rtd-theme 1.2.2 depends on docutils<0.19
```

but docutils 0.18 isn't python 3.11 compatible.

Reply:
I was able to install 0.18.1 though. Working though this. New error with `sphinx-data-error`

At a minimum we need to add `docutils <0.19`

Reply:
`sphinx-data-viewer` is trying to be installed by another package, but it is only compatible up to python 3.8 and is a dead project. 

We need to find that  package and delete it from our deps

Reply:
> and is a dead project.

Works for me! What kind of error message do you get on install?

```
(armi311) nick@orion:~/repos/armi$ python -V
Python 3.11.4
(armi311) nick@orion:~/repos/armi$ pip list |grep data-viewer
sphinx-data-viewer                0.1.2
```

Reply:
> I was able to install 0.18.1 though. Working though this. New error with `sphinx-data-viewer`
> 
> At a minimum we need to add `docutils <0.19`

I'll just pin it to 0.18.1 to be precise.

Reply:
> > and is a dead project.
> 
> Works for me! What kind of error message do you get on install?
> 
> ```
> (armi311) nick@orion:~/repos/armi$ python -V
> Python 3.11.4
> (armi311) nick@orion:~/repos/armi$ pip list |grep data-viewer
> sphinx-data-viewer                0.1.2
> ```

Here's the error I see in git bash on windows:

![MicrosoftTeams-image (21)](https://github.com/terrapower/armi/assets/5347433/d13172a1-f775-4fee-885e-6f3120ec3cbd)

Reply:
I cannot figure out what that error is coming from. maybe we can screenshare or something